src/dualscreen.tsx

```diff
@@ -558,7 +558,7 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
               this._setDefaultMode();
               this._setMode();
             });
-            this.secondaryKalturaPlayer.loadMedia({entryId, ks: this._player.config.session.ks});
+            this.secondaryKalturaPlayer.loadMedia({entryId, ks: this.player.config.session.isAnonymous ? '' : this.player.config.session.ks});
```

Comment:
First - I think the correct way of taking the session is from this.player.session and not through the config.
Second - I think maybe we need to change the approach...
If an anonymous ks should not be used outside of the provider then lets not expose it in case of anonymous.
Instead of having isAnonymous. We can just return an empty string in player.sessions.ks when ks is a anonymous.
What do you think? 
I am saying this because today it's dual screen and tomorrow it will be another plugin that will use the ks and will forget to handle the "anonymous" case

Reply:
@OrenMe @yairans @lianbenjamin

Reply:
For first - there is no `this.player.session` getter.
For second - the ks, either anonymous or not, reported by analytics, I'm not sure we can remove it @OrenMe wdyt?

Reply:
Yes, you are right. Don't know why I remembered that there was one.
And regarding the second I guess you are also right... :)

Reply:
So we can expose the KS but for doRequest I suggest we don't send ks and handle it internally in providers

Reply:
you can't handle it internally in providers, the secondary's provider is not the primary's one

Reply:
I hate when you right again